### ISSUE TITLE
Refresh doesn't redraw the screen (should it?)

### ISSUE THREAD

**Issue by glguy** [coreyoconnor/vty#104]
`refresh` is documented as

```
Refresh the display. nextEvent will refresh the display if a resize occurs. If other programs output to the terminal and mess up the display then the application might want to force a refresh.
```

I was hoping that this operation would attempt to redraw the whole screen, but as far as I can tell it just doesn't do anything. Should it do something?

My test case where I mash `p` a few times and then hope that `l` fixes things:

```haskell
import Control.Exception (bracket)
import Graphics.Vty

main :: IO ()
main =
  bracket (mkVty mempty) shutdown $ \vty ->
    do update vty (picForImage (string defAttr "some string"))
       loop vty 

loop :: Vty -> IO ()
loop vty =
  do ev <- nextEvent vty 
     case ev of
       EvKey KEsc        _ -> return ()
       EvKey (KChar 'p') _ -> putStrLn "bad news" >> loop vty 
       EvKey (KChar 'l') _ -> refresh vty         >> loop vty
       _                   -> loop vty
```

Should that have worked? Is there some other way to get the redraw effect?

**Comment by jtdaugherty** [comment]
This has been fixed in the associated commit.

**Comment by glguy** [comment]
Thank you so fixing that so quickly! I'm very excited to have my `^L` binding working.

**Comment by jtdaugherty** [comment]
You're welcome. It's now released as `5.8.1`.
